lib/helpers/wdio.conf.ejs

```diff
@@ -295,6 +295,42 @@ exports.config = {
     // afterSuite: function (suite) {
     // },
     /**
+     * Runs before a Cucumber feature
```

Comment:
Can we check if the framework is Cucumber and only include these when it is (and also not include `beforeTest` and other Mocha/Jasmine like hooks